tools/extensionserver/convert.go

```diff
@@ -0,0 +1,78 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package extensionserver
+
+import (
+	"io/ioutil"
+
+	core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	wasm "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/wasm/v3"
+	v3 "github.com/envoyproxy/go-control-plane/envoy/extensions/wasm/v3"
+	ptypes "github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/wrappers"
+)
+
+// Convert to an envoy config.
+// It so happens that 1.7 and 1.8 match in terms protobuf bytes, but not JSON.
```

Comment:
Which part is different? The additional StringValue wrapping?

Reply:
vm_config got renamed to inline_vm_config.